test-lib-openshift.sh

```diff
@@ -94,6 +94,27 @@ function ct_os_wait_pod_ready() {
   while ! ct_os_check_pod_readiness "${pod_prefix}" "true" ; do
     echo -n "."
     [ ${SECONDS} -gt ${timeout} ] && echo " FAIL" && return 1
+    SECONDS=$((SECONDS+3))
+    sleep 3
+  done
+  echo " DONE"
+}
+
+# ct_os_wait_rc_ready [pod_prefix, timeout]
+# --------------------
+# Wait maximum [timeout] for the rc having desired number of replicas ready.
+# Arguments: pod_prefix - prefix of the replication controller
+# Arguments: timeout - how many seconds to wait seconds
+function ct_os_wait_rc_ready() {
+  local pod_prefix="${1}" ; shift
+  local timeout="${1}" ; shift
+  SECONDS=0
+  echo -n "Waiting for ${pod_prefix} pod becoming ready ..."
+  while ! test "$(oc get rc -o custom-columns=NAME:.metadata.name,DESIRED:status.replicas,READY:status.readyReplicas 2>/dev/null \
+                 | grep ^${pod_prefix} | awk '$2==$3 {print "ready"}')" == "ready" ; do
```

Comment:
I prefer all shell vars quoted (except for in `case $var in` and assignments `foo=$bar`)...  But I think we don't have to block this PR with this ... let's go with shell-style fixes in different PR..

Reply:
Also this awk could be used:
```awk
BEGIN { exit_status=1; }                                                                                              
/^<pod_prefix>.*/ && $2 == $3 { exit_status=0 ; }                                                                                
END { exit exit_status; }     
```

Reply:
Double quotes added. If you prefer  awk solution I can change it.